organize_flowcell.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import argparse
+from glob import glob
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Organize flowcell for a specific project in /proj/ngi2016001/nobackup/NGI/DATA/<project>')
+    parser.add_argument('--runfolder', type=str, required=True, help='Name of runfolder (required)')
+    parser.add_argument('--project', type=str, required=True, help='Project name (required)')
+    parser.add_argument('--exclude_lane', type=str, required=False, default='', help='Lane number(s) to exclude (optional).')
+    parser.add_argument('--exclude_sample', type=str, required=False, default='', help='Sample name(s) to exclude (optional). Sample-1,Sample-2..')
```

Comment:
Generally I think you format your code very well, but some lines are bit long. I would recommend using a tool like [black](https://github.com/psf/black) to automatically format your code according to best practice standards.

Reply:
Thanks! I tried reformatting the code with black now.